lsp/nls/src/requests/completion.rs

```diff
@@ -106,7 +150,7 @@ fn get_identifier_before_field(text: &str) -> Option<String> {
         .rev()
         .collect();
 
-    get_identifier_before_dot(&text[..])
+    get_identifier_path(&text[..])
```

Comment:
Personally I'd find `text.as_str()` slightly clearer here, but it does essentially the same thing.